bdb/drivers/mysql.go

```diff
@@ -148,7 +148,7 @@ func (m *MySQLDriver) Columns(schema, tableName string) ([]bdb.Column, error) {
 				(select count(*) from information_schema.key_column_usage where table_schema = kcu.table_schema and table_name = tc.table_name and constraint_name = tc.constraint_name) = 1
 		) as is_unique
 	from information_schema.columns as c
-	where table_name = ? and table_schema = ?;
+	where table_name = ? and table_schema = ? and c.extra not like '%VIRTUAL%';
```

Comment:
@aarondl This fix does not seem to work for `GENERATED ALWAYS … STORED` columns, which are supported in MySQL[1] and do not include the keyword `VIRTUAL` in the column type.

1: https://dev.mysql.com/doc/refman/5.7/en/create-table-generated-columns.html